coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -594,6 +594,15 @@ func (s *server) acquireProtoJob(ctx context.Context, job database.ProvisionerJo
 			})
 		}
 
+		roles, err := s.Database.GetAuthorizationUserRoles(dbauthz.AsSystemRestricted(ctx), owner.ID)
+		if err != nil {
+			return nil, failJob(fmt.Sprintf("get owner authorization roles: %s", err))
+		}
+		ownerRbacRoles := []*sdkproto.OwnerRbacRoles{}
+		for _, role := range roles.Roles {
+			ownerRbacRoles = append(ownerRbacRoles, &sdkproto.OwnerRbacRoles{Name: role, OrgId: s.OrganizationID.String()})
```

Comment:
I think we should convert the nil uuid to the empty string. Since we losing the parsed `uuid`, determining a nil uuid is trickier on the other side of the communication. Just pass the empty string to indicate "no org id"

```suggestion
			if s.OrganizationID == uuid.Nil {
				ownerRbacRoles = append(ownerRbacRoles, &sdkproto.OwnerRbacRoles{Name: role, OrgId: ""})
				continue
			}
			ownerRbacRoles = append(ownerRbacRoles, &sdkproto.OwnerRbacRoles{Name: role, OrgId: s.OrganizationID.String()})
```